projects/medical/2d_image/microscopy_images/bactteria_detection/tools/prepare_dataset.py

```diff
@@ -0,0 +1,37 @@
+import glob
+import os
+import shutil
+
+from PIL import Image
+from sklearn.model_selection import train_test_split
+
+root_path = 'data/bactteria_detection/'
```

Comment:
```suggestion
root_path = 'data/'
```
Am I right?

Reply:
yes, the folder structure had changed, and the input data path in old script went wrong.